### PR TITLE
feat(columbus): update columbus api to new patch api

### PR DESCRIPTION
**Pull request by scortier**

### CODE REVIEW

File: plugins/sinks/columbus/columbus.go [hunk 1]
```diff
@@ -1,20 +1,26 @@
 package columbus
 
 type Record struct {
```
Comment (StewartJingga) [inline plugins/sinks/columbus/columbus.go:3]: we can rename this to be more clear e.g. `RequestPayload`. wdyt?

File: plugins/sinks/columbus/sink_test.go [hunk 2]
```diff
@@ -39,14 +38,10 @@ func TestInit(t *testing.T) {
...
 			},
-			{
-				"host": host,
-				"type": "",
-			},
 		}
 		for i, config := range invalidConfigs {
 			t.Run(fmt.Sprintf("test invalid config #%d", i+1), func(t *testing.T) {
-				columbusSink := columbus.New(newMockHTTPClient(config, http.MethodGet, url, []columbus.Record{}), testUtils.Logger)
+				columbusSink := columbus.New(newMockHTTPClient(config, http.MethodGet, url, columbus.Record{}), testUtils.Logger)
```
Comment (StewartJingga) [inline plugins/sinks/columbus/sink_test.go:44]: it should be `PATCH` instead of `GET`

File: plugins/sinks/columbus/sink_test.go [hunk 3]
```diff
@@ -313,33 +339,43 @@ func TestSink(t *testing.T) {
...
 			},
 		},
 	}
 
 	for _, tc := range successTestCases {
 		t.Run(tc.description, func(t *testing.T) {
-			tc.expected.Data = tc.data
-			payload := []columbus.Record{tc.expected}
+			tc.expected.Asset.Data = tc.data
+			fmt.Println(tc.expected)
```
Comment (StewartJingga) [inline plugins/sinks/columbus/sink_test.go:371]: dont forget to remove this

**Review by StewartJingga — CHANGES_REQUESTED** [review]

File: plugins/sinks/columbus/columbus.go (hunk 1 above)
Comment (scortier) [reply plugins/sinks/columbus/columbus.go:3]: `Record` specify more about what actually model is about as it is known to be a payload only, while `RequestPayload` is a general term for payload request for any api . So should we change it ?

Comment (StewartJingga) [reply plugins/sinks/columbus/columbus.go:3]: yes that worked before, but now Columbus does not have Record and only has Asset.

**Review by StewartJingga — APPROVED** [review]
LGTM

**Pull request merged**